content/doc/upgrade-guide/2.73.adoc

```diff
@@ -36,12 +36,22 @@ Possible workarounds:
 . Override it for the process running Jenkins, for example using the `-Duser.home=/path/to/directory` parameter to the `java` command, pointing to a writable directory.
 
 
-==== Known Issue: Possible connectivity issue involving ed25519 SSH keys
+==== Known Issue: Randomly occurring connection failures with password protected ed25519 SSH keys
 
 link:https://issues.jenkins-ci.org/browse/JENKINS-46754[JENKINS-46754]
 
-A possible regression involving SSH authentication using password protected ed25519 keys has been identified.
-More details and the current state of the investigation is available in the linked issue.
+A regression involving authentication using passphrase-protected ed25519 SSH keys has been identified in 2.73.
+It randomly causes failures in features using these keys, such as:
+
+. SSH Slaves Plugin connections
+. Accessing Git repositories with the JGit implementation
+. Accessing Subversion repositories (unconfirmed)
+
+Workarounds for this issue:
```

Comment:
I suppose in the case of an unpacked WAR (deployed to Tomcat, say, rather than using Winstone) you could simply delete `WEB-INF/lib/jbcrypt-0.4.jar` from disk. Untested and this is probably going further than we want.

Reply:
I wish there were a better way to describe the nature of the failure than "randomly occurring connection failures".  It is accurate to say that the failures were random, but the randomness was from one machine to the next, not from one execution of Jenkins to the next on the same machine.

If a master was showing the failure, it consistently showed the failure.  If it was not showing the failure, then it consistently did not show the failure.

Reply:
Perhaps due to the directory listing order on that system.